core/executor.py

```diff
@@ -189,7 +189,8 @@ def _MakeProcess(self, node, inherit_errexit=True):
                                         node,
                                         self.trap_state,
                                         self.multi_trace,
-                                        inherit_errexit=inherit_errexit)
+                                        inherit_errexit,
+                                        inherit_errtrace)
```

Comment:
Or is it possible to pass `self.exec_opts.errtrace()` here directly, instead of having all the params?  I think it's always the same value?